src/App.js

```diff
@@ -99,7 +103,8 @@ function App(props) {
         },
         config: {
           defaultFinality: undefined,
-        },
+          ...(props.rpc ? { nodeUrl: props.rpc } : {})
+        }
```

Comment:
This is correct, although let's make it simpler with a default RPC. I like lava network as default.
We should also make this dependent on networkId (testnet or mainnet).

nodeUrl: props.rpc || (networkId === "mainnet" ? "https://near.lava.build" : "https://near-testnet.lava.build")